src/core/mesh/qgsmeshrenderersettings.h

```diff
@@ -115,6 +114,17 @@ class CORE_EXPORT QgsMeshRendererScalarSettings
       NeighbourAverage,
     };
 
+    /**
+     *  This enumerator describes the extent used to compute min/max values
+     */
+    enum class MinMaxValueType
```

Comment:
Can you synchronize the naming here with those used in https://github.com/qgis/QGIS/pull/59644, and move this enum to Qgis accordingly?